modules/openapi-generator/src/test/java/org/openapitools/codegen/dart/DartModelTest.java

```diff
@@ -371,13 +371,13 @@ public static Object[][] varNames() {
                 {"FOO_BAR", "FOO_BAR"},
                 {"FOO_BAR_BAZ_", "FOO_BAR_BAZ_"},
                 {"123hello", "n123hello"},
-                {"_hello", "hello"},
+                {"_hello", "hello_"},
                 {"_double", "double_"},
-                {"_123hello", "n123hello"},
-                {"_5FOO", "n5fOO"},
-                 {"_FOO", "FOO"},
-                {"_$foo", "dollarFoo"},
-                {"_$_foo_", "dollarFoo"},
+                {"_123hello", "n123hello_"},
+                {"_5FOO", "n5fOO_"},
+                 {"_FOO", "FOO_"},
+                {"_$foo", "dollarFoo_"},
+                {"_$_foo_", "dollarFoo_"},
```

Comment:
We still have potential collisions here and we also loose double (multiple) underscores due to the implementation of the `camelize` method. But I am not sure it is worth to rewrite it all.